pkg/log/gin_logger.go

```diff
@@ -0,0 +1,67 @@
+package log
```

Comment:
We should add Apache LICENSE header. (you can copy from other files.

Reply:
> Simply run `make update-license && make update-gofmt` to update license header and imports.

Thanks!

Reply:
Fixed, thanks for your review.